mining.go

```diff
@@ -108,6 +108,43 @@ func (pq *txPriorityQueue) SetLessFunc(lessFunc txPriorityQueueLessFunc) {
 	heap.Init(pq)
 }
 
+// compareStakePriority compares the stake priority of two transactions.
+// It uses votes > tickets > regular transactions or revocations. It
+// returns 1 if i > j, 0 if i == j, and -1 if i < j in terms of stake
+// priority.
+func compareStakePriority(i, j *txPrioItem) int {
+	// Stake priority is assigned below as 2 for votes (highest),
+	// followed by 1 for tickets (2nd highest), followed by 0 for
+	// regular transactions and revocations (lowest).
+	votePriority := 2
```

Comment:
Please use constants for these.  There is no reason to create local variables on the stack for it!